### DISCUSSION TITLE
SMS Gateway

### DISCUSSION THREAD

**Discussion by grafst** [42wim/matterbridge#1956] [New bridges suggestions]
An sms gateway would be cool!

**Comment by VincentSC** [comment]
First problem is that each user is a mobile number, which makes it more difficult for chatting. If you want to have one single number for a channel, then this could possibly be bridged. Not for group-chats.

Second problem is the many platforms to use, which all have their own API and interface. This means there needs to be a generic interface that handles this, and uses plugins for each router, mobile phone and web-service existing. See for example https://github.com/Salamek/huawei-lte-api

Can you describe your usecase with as many details as possible?

**Comment by Labfox** [comment]
I think this would be doable with something like [textbee](https://github.com/vernu/textbee) or [httpsms](https://github.com/NdoleStudio/httpsms). I had an implementation some time ago of this with my own bridge system. However, the resources needed to run an sms gateway are way bigger than for a normal bridge.
